google-cloud-apis/java-apis/maven-apis/src/com/google/cloud/tools/intellij/cloudapis/maven/BomComboBox.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.cloudapis.maven;
+
+import com.google.common.collect.Lists;
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.ListCellRendererWrapper;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import javax.swing.JComboBox;
+import javax.swing.JList;
+
+/**
+ * Combobox that populates list of available BOM versions and selects currently active BOM version
+ * from the given project/module.
+ */
+public class BomComboBox extends JComboBox<String> {
```

Comment:
just fyi that this on potentially will span more than just maven (gradle for example can support BOMs directly in the latest versions).

Reply:
initially I think its ok to just have it for Maven though

Reply:
I agree, this looks like a more generic class - but still kind of coupled to Maven services and classes for now. We can always split this and use for Gradle too later.